### ISSUE TITLE
Issue: Get logs and version

### ISSUE THREAD

**Issue by greenszpila** [newrelic/docs-website#3183]
### Tell us what you need

The current instruction refers to K8's with a manifest file.
Could you provide an instruction to gather verbose logs for K8s Integration installed via Helm ?

```
Enable verbose logging:
helm upgrade -n <namespace> --reuse-values newrelic-bundle --set newrelic-infrastructure.verboseLog=true newrelic/nri-bundle
Gather logs from the pod connecting to KSM with these instructions:
https://docs.newrelic.com/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version/#logs-pod-kubestatemetrics
Disable verbose logging:
helm upgrade --reuse-values newrelic-bundle --set newrelic-infrastructure.verboseLog=false newrelic/nri-bundle
```

### Anything else you'd like to share?

Add other context like screenshots, links to other docs, and information about
your environment (operating system, application framework, etc.).

### Doc information (don't delete this section)

* https://docs.newrelic.com/docs/integrations/kubernetes-integration/troubleshooting/get-logs-version/
* **OS**: Unknown
* **Browser**: Unknown
* **Device**: Unknown

**Comment by urbiz-nr** [comment]
Confirmed this has been filed by a relic. Ping me if you need to get in touch with him!

**Comment by mmfred** [comment]
Thanks @greenszpila - I requested review from the Kubernetes experts. I'll update the docs once I hear back.

**Comment by zuluecho9** [comment]
I think this issue will be solved with this PR https://github.com/newrelic/docs-website/pull/3252, is that correct @mmfred ?

**Comment by mmfred** [comment]
> I think this issue will be solved with this PR #3252, is that correct @mmfred ?

yes. Finalizing some changes.
